packages/@okta/vuepress-site/docs/reference/api/asa/clients/index.md

```diff
@@ -0,0 +1,263 @@
+| `user_name`   | string | The ASA User to whom this ASA Client belongs. |
+
+#### Usage example
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Authorization: Bearer ${jwt}" \
+https://app.scaleft.com/v1/teams/${team_name}/clients/${client_id}
+```
+
+##### Response
+
+```json
+{
+	"deleted_at": null,
+	"description": "Work laptop",
+	"encrypted": true,
+	"hostname": "LightInAugust",
+	"id": "9c199afe-0ca5-427a-baac-c4341707d82b",
+	"os": "macOS 10.14.6",
+	"state": "PENDING",
+	"user_name": "Jason.Compson.IV"
+}
+```
+### Revoke an ASA Client
+
+<ApiOperation method="DELETE" url="https://app.scaleft.com/v1/teams/${team_name}/clients/${client_id}" />
+Revoke an ASA Client's access to this Team
+
+This endpoint requires one of the following roles: `authenticated_client`, `access_user`, `access_admin`.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `client_id`   | string | The UUID of the Client. |
+| `team_name`   | string | The name of your Team. |
+
+
+#### Request query parameters
+
+This endpoint has no query parameters.
+
+#### Request body
+
+This endpoint has no request body.
+
+#### Response body
+This endpoint returns a `204 No Content` response on a successful call.
+
+
+#### Usage example
+
+##### Request
+
+```bash
+curl -v -X DELETE \
+-H "Authorization: Bearer ${jwt}" \
+https://app.scaleft.com/v1/teams/${team_name}/clients/${client_id}
+```
+
+##### Response
+
+```json
+HTTP 204 No Content
+```
+### Approve or assign an ASA Client
+
+<ApiOperation method="PUT" url="https://app.scaleft.com/v1/teams/${team_name}/clients/${client_id}" />
+This endpoint approves or assigns an ASA Client to be used by an ASA User. Used for ASA Clients enrolled using a token policy or a policy that requires approval
+
+This endpoint requires one of the following roles: `access_admin`.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `client_id`   | string | The UUID of the Client. |
+| `team_name`   | string | The name of your Team. |
+
+
+#### Request query parameters
+
+This endpoint has no query parameters.
+
+#### Request body
+
+This endpoint requires an object with the following fields.
+| Properties | Type        | Description          |
+|----------|-------------|----------------------|
+| `state`   | string | The state of the ASA Client. One of: `ACTIVE`, `PENDING`, or `DELETED`. |
+| `user_name`   | string | The ASA User to whom this ASA Client belongs. |
+
+#### Response body
+This endpoint returns a `204 No Content` response on a successful call.
+
+
+#### Usage example
+
+##### Request
+
+```bash
+curl -v -X PUT \
+-H "Authorization: Bearer ${jwt}" \
+--data '{
+	"state": "ACTIVE",
+	"user_name": "Jason.Compson.IV"
+}' \
+https://app.scaleft.com/v1/teams/${team_name}/clients/${client_id}
```

Comment:
is this URL still accurate? Checking as it is going to `scaleft.com`

Reply:
This URL is still accurate.